nova/templates/etc/_hypervisor-ironic.conf.tpl

```diff
@@ -0,0 +1,12 @@
+{{- define "ironic_conf" -}}
+{{- $context := index . 0 -}}
```

Comment:
It seems we are not using the global $context in this template so we could avoid the tuple foo in this case, no?